package.json

```diff
@@ -39,7 +39,7 @@
   ],
   "license": "Apache-2.0",
   "devDependencies": {
-    "eslint": "^8.16.0",
-    "mocha": "^10.0.0"
+    "eslint": "^8.25.0",
```

Comment:
Please update the `eslint@8.26.0` in package.json and re-create npm-shrinkwrap.json.